plugins/org.jkiss.dbeaver.net.ssh.ui/src/org/jkiss/dbeaver/ui/net/ssh/SSHTunnelDefaultConfiguratorUI.java

```diff
@@ -742,6 +733,33 @@ public void widgetSelected(SelectionEvent e) {
             }
         }
 
+        @NotNull
+        private ConfigurationFileSelector getConfigurationFileSelector(@NotNull ModifyListener listener) {
+            boolean isDistributed = DBWorkbench.isDistributed();
+            final ConfigurationFileSelector privateKeyText;
+            privateKeyText = new ConfigurationFileSelector(
```

Comment:
```suggestion
            ConfigurationFileSelector privateKeyText = new ConfigurationFileSelector(
```

And maybe also rename to something else, like `selector`:

```suggestion
            ConfigurationFileSelector selector = new ConfigurationFileSelector(
```